databases/levelup.md

```diff
@@ -0,0 +1,151 @@
+# LevelUP
+
+[feathers-levelup](https://github.com/feathersjs/feathers-levelup) is a database adapter for the fast and simple [LevelDB](https://github.com/google/leveldb), though many other backing stores are supported by supplying alternate backing stores to your [LevelUP](https://github.com/Level/levelup) instances. 
+
+LevelUP backing store &rarr; LevelUP instance &rarr; Feathers service
+
+LevelDB is a key-value database that stores its keys in lexicographical order, allowing for efficient range queries over keys. Because LevelDB can only stream results in the order they are stored, avoid using `$sort` in your `find` calls, as the entire keyspace may be loaded in order to perform an in-memory sort. Read further to learn about ordering your LevelDB keys and designing your application around efficient range queries.
+
+LevelUP currently supports the following backing stores:
+
+- LevelDB
+- Amazon DynamoDB
+- AsyncStorage
+- Basho's LevelDB Fork
+- Google Sheets
+- localStorage
+- In-memory LRU Cache
+- IndexedDB
+- JSON Files
+- Knex (sqlite3, postgres, mysql, websql)
+- Medea
+- Memory
+- MongoDB
+- MySQL
+- Redis
+- Riak
+- RocksDB
+- Windows Azure Table Storage
+
+## Installation
+
+```bash
+npm install levelup leveldown feathers-levelup --save
+```
+
+## Getting Started
+
+Creating a LevelUP service:
+
+```js
+const levelup = require('levelup');
+const levelupService = require('feathers-levelup');
+
+// this will create a database on disk under ./todos
+const db = levelup('./todos', { valueEncoding: 'json' });
+
+app.use('/todos', levelupService({ db: db }));
+```
+
+See the [LevelUP Guide](https://github.com/Level/levelup) for more information on configuring your database, including selecting a backing store.
+
+## Key Order
+
+By default, LevelDB stores records on disk [sorted by key](http://leveldb.org/). Storing sorted keys is one of the distinguishing features of LevelDB, and the LevelUP interface is designed around it.
+
+When records are created, a key is generated based on a the value of `options.sortField`, plus a uuid. By default, `_createdAt` is automatically set on each record and its value is prepended to the key (id).
+
+Change the `sortField` option to the field of your choice to configure your database's key ordering:
+
+```js
+app.use('todos', service({
```

Comment:
I'll probably just convert these to `messages` later in order to keep things consistent with the guide and other places in the docs.